pkgs/applications/video/vlc/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, autoreconfHook
+{pkgs, lib, stdenv, fetchurl, autoreconfHook
```

Comment:
Please do not pass pkgs in as an input but the individual packages. In this case it would be ``ibXpm freetype libXext libXinerama``.

Reply:
I think the author can also use `xorg` that is already passed here too: https://github.com/NixOS/nixpkgs/pull/109235/files#diff-e0d4a136479523bc5cfaf6a3bf53dcdb52dd51911f7fbaf199a0136e8e49371fL2